js/shape.js

```diff
@@ -54,7 +57,7 @@ class Shape {
             canvas.shapes[shapeId].showNodes();
```

Comment:
is there no onclick we can use for this shape conveniently? can you explain

Reply:
I didn't understand :(